From: "M.H.VanLeeuwen" 
Subject: crosstool, sparc64, linux 2.6.4, gcc 3.3.3, glibc 2.3.2 success
Date: Mon, 15 Mar 2004 00:23:04 -0600
To: crossgcc@sources.redhat.com
Message-ID: <40554BC8.749A5EC3@megsinet.net>
...

backported from GLIBC CVS

Fixes:
../sysdeps/unix/sysv/linux/sparc/sparc64/pause.c:1:39: sysdeps/unix/common/pause.c: No such file or directory
make[2]: *** [/cross/build/sparc64-unknown-linux-gnu/gcc-3.3.2-glibc-2.3.2/build-glibc/posix/pause.o] Error 1
make[2]: Leaving directory `/cross/build/sparc64-unknown-linux-gnu/gcc-3.3.2-glibc-2.3.2/glibc-2.3.2/posix'
make[1]: *** [posix/subdir_lib] Error 2
make: *** [all] Error 2

=========================================================================================================================
--- glibc-2.3.2/sysdeps/unix/sysv/linux/sparc/sparc64/pause.c.orig	Sun Mar  7 08:51:20 2004
+++ glibc-2.3.2/sysdeps/unix/sysv/linux/sparc/sparc64/pause.c	Sun Mar  7 08:51:29 2004
@@ -1 +1 @@
-#include <sysdeps/unix/common/pause.c>
+#include <sysdeps/posix/pause.c>
